.github/workflows/generate.yaml

```diff
@@ -30,9 +30,12 @@ jobs:
           repository: googleapis/discovery-artifact-manager
           fetch-depth: 1
           path: discovery-artifact-manager
-      - uses: actions/setup-python@v4
-        with:
-          python-version: 2.7.18
+      - run: |
+          sudo apt update
+          sudo apt install python2
+          python2 --version

```

Comment:
Can we echo something like "Using Python version 2.x"?

Reply:
replaced with `echo` message